### PR TITLE
updated README for v0.95 release

### PR DESCRIPTION
**Pull request by rsnancollas**

### CODE REVIEW

**Comment by rsnancollas** [comment]
This was from the script

File: support-docs/README.md [hunk 1]
```diff
@@ -30,1128 +30,1091 @@ This document is intended for digital datasheet producers such as components ven
 ### 2. Use Cases 
 Digital datasheets can be used for many applications. The list of use cases included here is not meant to be exhaustive and it is expected that new applications will be developed as more people start using these digital datasheets. Sample applications include automated hardware design checks to identify bugs earlier in the design cycle, automated hardware designs to speed up board development, components comparison to identify replacement components on a design. 
 
-### 3.Specifications
+### 3.1	 Common
 
-#### 3.1 Digital Datasheet Format
-A digital datasheet shall be written in the JSON (JavaScript Object Notation) format. The specification is written in JSON schema to facilitate validation of a JSON datasheet.
```
Comment (Carrotman42) [inline support-docs/README.md:36]: Do we not want this top-level description somewhere still?

**Review by Carrotman42 — APPROVED** [review]
Other than my one comment, lgtm

File: support-docs/README.md (hunk 1 above)
Comment (rsnancollas) [reply support-docs/README.md:36]: Good call! I missed this and just fixed it

**Pull request merged**